agent/config/config_windows.go

```diff
@@ -0,0 +1,62 @@
+// +build windows
+// Copyright 2014-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package config
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/aws/amazon-ecs-agent/agent/engine/dockerclient"
+	"github.com/aws/amazon-ecs-agent/agent/utils"
+)
+
+const (
+	defaultCredentialsAuditLogFile = `log\audit.log`
+	httpPort                       = 80
+)
+
+// DefaultConfig returns the default configuration for Windows
+func DefaultConfig() Config {
+	programData := utils.DefaultIfBlank(os.Getenv("ProgramData"), `C:\ProgramData`)
+	ecsRoot := filepath.Join(programData, "Amazon ECS")
+	return Config{
+		DockerEndpoint:              "npipe:////./pipe/docker_engine",
+		ReservedPorts:               []uint16{DockerReservedPort, DockerReservedSSLPort, AgentIntrospectionPort, AgentCredentialsPort},
+		ReservedPortsUDP:            []uint16{},
+		DataDir:                     filepath.Join(ecsRoot, "data"),
+		DisableMetrics:              false,
+		ReservedMemory:              0,
+		AvailableLoggingDrivers:     []dockerclient.LoggingDriver{dockerclient.JsonFileDriver},
+		TaskCleanupWaitDuration:     DefaultTaskCleanupWaitDuration,
+		DockerStopTimeout:           DefaultDockerStopTimeout,
+		CredentialsAuditLogFile:     filepath.Join(ecsRoot, defaultCredentialsAuditLogFile),
+		CredentialsAuditLogDisabled: false,
+		ImageCleanupDisabled:        false,
```

Comment:
disabled = false means that it is enabled by default.  There's a bit of technical debt in our config package that means our defaults must be false in order to be overridden, so anything that is enabled by default has a config field for "disabled" instead of enabled.

tl;dr: Image cleanup is enabled by default on both Windows and Linux.